pennylane/circuit_drawer/__init__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This subpackage provides the circuit drawing functionality used to render QNodes
+as circuit diagrams.
+
+.. currentmodule:: pennylane.circuit_drawer
+.. autosummary::
+    :toctree: api
+
+    default_qubit
+    default_gaussian
```

Comment:
Would it be working with these two devices or was this part coming from another module docstring?

Reply:
Oh sorry, that was a copy-paste error!